modulemd/tests/test-modulemd-moduleindex.c

```diff
@@ -1186,6 +1186,100 @@ test_module_index_read_compressed (void)
 }
 
 
+static void
+test_module_index_read_def_dir (void)
+{
+  g_autoptr (ModulemdModuleIndex) idx = modulemd_module_index_new ();
+  g_autoptr (GError) error = NULL;
+  g_autofree gchar *path =
+    g_build_path ("/", g_getenv ("TEST_DATA_PATH"), "defaults", NULL);
+  g_autofree gchar *overrides_path =
+    g_build_path ("/", path, "overrides", NULL);
+  g_auto (GStrv) module_names = NULL;
+  g_autoptr (GHashTable) defaultdict = NULL;
+
+  g_assert_nonnull (idx);
+
+  /* First verify that it works without overrides */
+  g_assert_true (modulemd_module_index_update_from_defaults_directory (
+    idx, path, TRUE, NULL, &error));
+  g_assert_no_error (error);
+
+  /* There should be three modules in the index now:
+   * - meson
+   * - ninja
+   * - nodejs
+   */
+  module_names = modulemd_module_index_get_module_names_as_strv (idx);
+  g_assert_nonnull (module_names);
+
+  g_assert_cmpint (g_strv_length (module_names), ==, 3);
+
+  g_strv_contains ((const gchar *const *)module_names, "meson");
```

Comment:
```suggestion
  g_assert_true (g_strv_contains ((const gchar *const *)module_names, "meson"));
```
And the next two lines...